### DISCUSSION TITLE
Pythonic plugin API?

### DISCUSSION THREAD

**Discussion by Destroyinator69420** [Tribler/tribler#6212] [Ideas]

**Comment by hbiyik** [comment]
something like this?
https://github.com/hbiyik/repository.biyik/tree/master/plugin.program.tribler/tribler/api

**Reply by sjehuda** [reply]
Is there a documentation to the plugin system?

**Reply by hbiyik** [reply]
nah and it is 4 years old and most likely does not work anymore

**Reply by sjehuda** [reply]
Thanks.

I would be glad to write an XMPP plugin to integrate with Tribler as a contact bot functioning as a remote control.

Once there be current documentation.

https://github.com/Tribler/tribler/issues/8358
